mobile-app/app/screens/AppNavigator/screens/Loans/LoansScreen.tsx

```diff
@@ -166,7 +165,7 @@ export function LoansScreen ({ route }: Props): JSX.Element {
     >
       <Tabs tabSections={tabsList} testID='loans_tabs' activeTabKey={activeTab} />
       {activeTab === TabKey.YourVaults && <Vaults />}
-      {activeTab === TabKey.BrowseLoans && loadingState === 'loading'
```

Comment:
Currently it will always be the skeleton loader, maybe just add temporary solution or we can fix it when integrating loan api, both are fine to me

Reply:
Yes should be ok, it's just a static UI anyway